ou_dedetai/config.py

```diff
@@ -59,55 +60,55 @@
     globals()[key] = os.getenv(key, default)
 
 # Set other run-time variables not set in the env.
-ACTION = 'app'
-APPIMAGE_FILE_PATH = None
-authenticated = False
-BADPACKAGES = None
+ACTION: str = 'app'
+APPIMAGE_FILE_PATH: Optional[str] = None
+authenticated = False # FIXME: no references
+BADPACKAGES: Optional[str] = None # FIXME: no references
 DEFAULT_CONFIG_PATH = os.path.expanduser(f"~/.config/FaithLife-Community/{name_binary}.json")  # noqa: E501
-FLPRODUCTi = None
-GUI = None
-INSTALL_STEP = 0
-INSTALL_STEPS_COUNT = 0
+FLPRODUCTi: Optional[str] = None
+GUI = None # FIXME: no references
```

Comment:
This most likely can be safely removed. I think it would have been used to keep track of the GUI object, but we have the `app` var we pass around now.